src/util/azure_utils.ts

```diff
@@ -134,12 +144,11 @@ export async function requestAKVSignature(
     plaintext: Buffer, keyVaultCredentials: AzureKeyVaultCredentials) {
   try {
     const signResult = await sign(plaintext, keyVaultCredentials);
-
     if (!signResult.result) {
       throw new Error('Azure Key Vault Signed result empty');
     }
     return findEthereumSig(Buffer.from(signResult.result));
```

Comment:
should this be `calculateSignature`?

Reply:
This function finds r and s values from an ethereum signature.
We can make that change though.

Reply:
@ni3gavhane Do you want `findEthereumSig` to be changed to `calculateSignature`?

Reply:
it is not finding the Ethereum style signature, it retrieves the `r` and `s`. Hence I find `recoverSignature` more readable. What do you think?

Reply:
Makes sense, will go with `recoverSignature`